python/cudf/cudf/core/buffer.py

```diff
@@ -33,21 +33,20 @@ class Buffer(Serializable):
         object is kept in this Buffer.
     """
 
-    ptr: int
-    size: int
-    _owner: Any
+    _size: int
+    _ptr: int
+    _owner: object
 
     def __init__(
-        self, data: Any = None, size: Optional[int] = None, owner: Any = None
+        self, data: Any = None, size: int = None, owner: object = None
     ):
-
         if isinstance(data, Buffer):
-            self.ptr = data.ptr
-            self.size = data.size
+            self._size = data.size
+            self._ptr = data._ptr
             self._owner = owner or data._owner
         elif isinstance(data, rmm.DeviceBuffer):
-            self.ptr = data.ptr
-            self.size = data.size
+            self._size = data.size
+            self._ptr = data.ptr
```

Comment:
Let's be consistent about always assigning `_ptr` and then `_size`, as in the previous code.
```suggestion
            self._ptr = data.ptr
            self._size = data.size
```